tools/experiments/experiments.js

```diff
@@ -185,7 +185,7 @@ const EXPERIMENTS = [
   },
   {
     id: 'amp-animation',
-    name: 'High-performing keyframe animations in AMP.',
+    name: 'High-performing keyframe animations in AMP (launched).',
```

Comment:
we can remove the whole entry before updating `config` files not?

Reply:
Yeah, we can. I thought we prefer this. Do you have a strong opinion on this?

Reply:
no, was mostly curious since I did a similar change recently and removed the experiment entry, got worried that it may have a bad side effect.